__tests__/news.test.js

```diff
@@ -42,30 +59,29 @@ describe("GET", () => {
       });
   });
 
-  it("/api/articles/:article_id", () => {
+  test("/api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/cat")
       .expect(400)
       .then((res) => {
         const msg = res.body.err;
 
         expect(msg).toBe("Not an Article ID. Please enter numeric value.");
+      });
+  });
 
-  it("/api/topics", () => {
+  test("/api/users", () => {
     return request(app)
-      .get("/api/topics")
+      .get("/api/users")
       .expect(200)
-      .then((response) => {
-        const { topics } = response.body;
-
-        expect(Array.isArray(topics)).toBe(true);
-        expect(topics.length === 3).toBe(true);
+      .then((res) => {
+        const userList = res.body.users;
 
-        topics.forEach((topic) => {
-          expect(topic).toHaveProperty("slug", expect.any(String));
-          expect(topic).toHaveProperty("description", expect.any(String));
+        userList.forEach((user) => {
```

Comment:
This is a good test.  However, if we are using a loop like this to check each value inside it we have to check before that the array has a length > 0.  Otherwise, if we get an empty array back it would still pass the test (false positive)